e2e-test/java/io/cdap/plugin/slt/locators/SLTLocators.java

```diff
@@ -0,0 +1,110 @@
+ */
+
+package io.cdap.plugin.slt.locators;
+
+import io.cdap.e2e.utils.SeleniumDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+/**
+ * SLT Locators.
+ */
+public class SLTLocators {
+    private static WebDriver CDFDriver;
+
+    static {
+        CDFDriver = SeleniumDriver.getDriver();
+    }
+
+    @FindBy(how = How.XPATH, using = "//input[@data-cy=\"name\"]")
+    public static WebElement name;
+    @FindBy(how = How.XPATH, using = "//input[@data-cy=\"gcpProjectId\"]")
+    public static WebElement gcpProjectId;
+    @FindBy(how = How.XPATH, using = "//div[contains(text(),'SAP Landscape Transformation (SLT)')]")
+    public static WebElement sltplugin;
+    @FindBy(how = How.XPATH, using = "//input[@data-cy=\"gcsDataPath\"]")
+    public static WebElement gcsDataPath;
+    @FindBy(how = How.XPATH, using = "//input[@data-cy=\"guid\"]")
+    public static WebElement guid;
+    @FindBy(how = How.XPATH, using = "//input[@data-cy=\"massTransferId\"]")
+    public static WebElement massTransferId;
+    @FindBy(how = How.XPATH, using = "//input[@data-cy=\"sapJcoLibGcsPath\"]")
+    public static WebElement sapJcoLibGcsPath;
+    @FindBy(how = How.XPATH, using = "//input[@data-cy=\"jco.client.ashost\"]")
+    public static WebElement jcoclienthost;
+    @FindBy(how = How.XPATH, using = "//input[@data-cy=\"jco.client.client\"]")
+    public static WebElement jcoclient;
+    @FindBy(how = How.XPATH, using = "//input[@data-cy=\"jco.client.sysnr\"]")
+    public static WebElement jcoclientsysnr;
+    @FindBy(how = How.XPATH, using = "//input[@data-cy=\"jco.client.user\"]")
+    public static WebElement jcoclientuser;
+    @FindBy(how = How.XPATH, using = "//input[@data-cy=\"jco.client.passwd\"]")
+    public static WebElement jcoclientpasswd;
+    @FindBy(how = How.XPATH, using = "//*[contains(text(),'Next')]")
+    public static WebElement next;
+    @FindBy(how = How.XPATH, using = "//input[@data-cy=\"project\"]")
+    public static WebElement project;
+    @FindBy(how = How.XPATH, using = "//input[@data-cy=\"datasetName\"]")
+    public static WebElement datasetName;
+    @FindBy(how = How.XPATH, using = "//span[contains(text(),'Configure target')]")
+    public static WebElement configure;
+    @FindBy(how = How.XPATH, using = "//span[contains(text(),'Configure advanced properties')]")
+    public static WebElement configureadvance;
+    @FindBy(how = How.XPATH, using = "//span[contains(text(),'Review assessment')]")
+    public static WebElement review;
+    @FindBy(how = How.XPATH, using = "//span[contains(text(),'View summary')]")
+    public static WebElement viewsummary;
+    @FindBy(how = How.XPATH, using = "//span[contains(text(),'Deploy Replication Job')]")
+    public static WebElement deployPipeline;
+    @FindBy(how = How.XPATH, using = "//*[contains(text(), 'Logs')]")
```

Comment:
Create xpath with data-cy. Make use of other properties and attributes only if unique xpath can not be created 
 using data-cy.
applies to all.

Reply:
Tried to use data-cy tag as much possible for uniformity but for the tags I have used best possible option available.